### PR TITLE
Update Reports.swagger.json

### PR DESCRIPTION
**Pull request by wyattlrose**
JIRA
https://jira.concur.com/browse/ARCH-1337

After reviewing the swagger file we uploaded during the cost object endpoint externalization process we've spotted a few small issues this PR corrects. Reasoning added as comments to each change.

### CODE REVIEW

File: src/api-explorer/v4-0/Reports.swagger.json [hunk 1]
```diff
@@ -3,20 +3,13 @@
...
         "license": {
             "name": "SAP Concur",
             "url": ""
         },
         "version": "4.14.0"
     },
-    "externalDocs": {
-        "description": "Expense Report Service Readme",
-        "url": "https://github.concur.com/inception/expense-report-service"
-    },
```
Comment (wyattlrose) [inline src/api-explorer/v4-0/Reports.swagger.json:19]: We do not want a link to our repository shared publicly.

File: src/api-explorer/v4-0/Reports.swagger.json [hunk 2]
```diff
@@ -52,14 +45,14 @@
...
-        },
         {
             "name": "Allocations",
             "description": "Gets the allocations that belong to a specific expense on an expense report and updates an allocation on an existing expense in an expense report"
-        }
+        },
+      {
+        "name": "Cost Object Workflows",
+        "description": "Manages cost object workflow transitions for a report"
+      }
```
Comment (wyattlrose) [inline src/api-explorer/v4-0/Reports.swagger.json:55]: Cost Object Workflows tag was missing.

File: src/api-reference/expense/expense-report/v4.cost-objects.markdown [hunk 3]
```diff
@@ -167,10 +167,10 @@ curl --location --request GET https://integration.api.concursolutions.com/expens
 |Name|Type|Format|Description|
 |---|---|---|---|
 |`name`|`string`|-|**Required** The name of the cost object.
-|`approvedAmount`|`[Amount](#amount-schema)`|-|**Required** The approved amount of the cost object expense(s) in the report currency.|
-|`claimedAmount`|`[Amount](#amount-schema)`|-|**Required** The approved amount of the cost object expense(s) in the report currency.|
+|`approvedAmount`|[`Amount`](#amount-schema)|-|**Required** The approved amount of the cost object expense(s) in the report currency.|
+|`claimedAmount`|[`Amount`](#amount-schema)|-|**Required** The approved amount of the cost object expense(s) in the report currency.|
 |`approverId`|`String`|-|**Required** The unique identifier of the cost object approver.|
-|`expenses`|`[CostObjectExpense](#amount-schema)`|-|**Required** TA list of expenses associated with the cost object.|
+|`expenses`|[`CostObjectExpense`](#amount-schema]|-|**Required** TA list of expenses associated with the cost object.|
```
Comment (wyattlrose) [inline src/api-reference/expense/expense-report/v4.cost-objects.markdown:173]: Had broken the links with incorrect formatting, fixing that as well.

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**